### ISSUE TITLE
use mermaidAPI

### ISSUE THREAD

**Issue by piranna** [JozoVilcek/gitbook-plugin-mermaid#18]
mermaid [now has an offline render](http://knsv.github.io/mermaid/#api-usage) that directly returns a SVG image. By using it, we could be able to remove all the dependencies on PhantomJS and make the module a lot more simpler and faster :-)

**Comment by JozoVilcek** [comment]
I do not think so, that API assumes it runs in browser (or equivalent) which does heavy lifting in `svg` rendering. It is not replacing `mermaidCLI`
